charts/redpanda/templates/secrets.yaml

```
@@ -75,3 +76,17 @@ stringData:
         sleep 0.5
     done
 {{- end }}
+{{- if and .Values.auth.sasl.enabled ( gt ( len .Values.auth.sasl.users ) 0 ) }}
```

Should we have a value that allows the user to administer this secret outside of helm?

---

Not sure what that would get us, wouldnt just an empty list and just a secret referencing things be enough? It does require a bit more logic but i think its tolerable.

---

An empty list is false

```suggestion
{{- if and .Values.auth.sasl.enabled .Values.auth.sasl.users }}
```

---

alright